frontend/awx/administration/instances/InstanceForm.cy.tsx

```diff
@@ -75,12 +75,6 @@ describe('Add instance Form', () => {
       }
     );
 
-    cy.get('[data-cy="peer-select-form-group"]').within(() => {
-      cy.get('button').eq(1).click();
-    });
-    cy.get('[data-cy="checkbox-column-cell"]').click();
-    cy.clickButton('Confirm');
-
     cy.get('[data-cy="enabled"]').click();
```

Comment:
We'll want to omit interaction with the enable/disable toggle in our tests as it can have unintended effects with job runs.